### PR TITLE
Remove `B` type param: rest of axum

### PR DESCRIPTION
**Pull request by davidpdrsn**

### CODE REVIEW

File: axum/src/docs/middleware.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ let app = Router::new()
     .layer(layer_one)
     .layer(layer_two)
     .layer(layer_three);
-# let _: Router<(), axum::body::Body> = app;
+# let _: Router<()> = app;
```
Comment (jplatte) [inline axum/src/docs/middleware.md:100]:
```suggestion
# let _: Router = app;
```

File: axum/src/docs/middleware.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ let app = Router::new()
             .layer(layer_two)
             .layer(layer_three),
     );
-# let _: Router<(), axum::body::Body> = app;
+# let _: Router<()> = app;
```
Comment (jplatte) [inline axum/src/docs/middleware.md:159]:
```suggestion
# let _: Router = app;
```

File: axum/src/extract/multipart.rs [hunk 3]
```diff
@@ -284,6 +280,8 @@ mod tests {
     // No need for this to be a #[test], we just want to make sure it compiles
     fn _multipart_from_request_limited() {
         async fn handler(_: Multipart) {}
-        let _app: Router<(), http_body::Limited<Body>> = Router::new().route("/", post(handler));
+        let _app: Router<()> = Router::new()
```
Comment (jplatte) [inline axum/src/extract/multipart.rs:283]:
```suggestion
        let _app: Router = Router::new()
```

File: axum/src/routing/tests/mod.rs [hunk 4]
```diff
@@ -714,18 +713,18 @@ async fn limited_body_with_streaming_body() {
...
-        .send()
-        .await;
-    assert_eq!(res.status(), StatusCode::OK);
+    // let stream = futures_util::stream::iter(vec![Ok::<_, hyper::Error>("a".repeat(LIMIT))]);
+    // let res = client
+    //     .post("/")
+    //     .body(reqwest::Body::wrap_stream(stream))
+    //     .send()
+    //     .await;
+    // assert_eq!(res.status(), StatusCode::OK);
```
Comment (jplatte) [inline axum/src/routing/tests/mod.rs:722]: Was this supposed to be commited?

Comment (davidpdrsn) [reply axum/src/routing/tests/mod.rs]: Nope!

**Review by jplatte — APPROVED** [review]

**Pull request merged**